java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -0,0 +1,51 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.Ok;
+import glide.api.models.commands.SetOptions;
+import glide.api.models.commands.SetOptions.ConditionalSet;
+import glide.api.models.commands.SetOptions.SetOptionsBuilder;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * String Commands interface to handle single commands.
+ *
+ * @see <a href="https://redis.io/commands/?group=string">String Commands</a>
+ */
+public interface StringCommands {
+
+    /**
+     * Get the value associated with the given key, or null if no such value exists.
+     *
+     * @see <a href="https://redis.io/commands/get/">redis.io</a> for details.
+     * @param key The key to retrieve from the database.
+     * @return Response from Redis. <code>key</code> exists, returns the <code>value</code> of <code>
+     *     key</code> as a String. Otherwise, return <code>null</code>.
+     */
+    CompletableFuture<String> get(String key);
+
+    /**
+     * Set the given key with the given value.
+     *
+     * @see <a href="https://redis.io/commands/set/">redis.io</a> for details.
+     * @param key The key to store.
+     * @param value The value to store with the given <code>key</code>.
+     * @return Response from Redis.
+     */
+    CompletableFuture<Ok> set(String key, String value);
+
+    /**
+     * Set the given key with the given value. Return value is dependent on the passed options.
+     *
+     * @see <a href="https://redis.io/commands/set/">redis.io</a> for details.
+     * @param key The key to store.
+     * @param value The value to store with the given key.
+     * @param options The Set options.
+     * @return Response from Redis containing a <code>String</code> or <code>null</code> response. The
```

Comment:
"Response from Redis containing a <code>String</code> or <code>null</code> response." I think it doesn't describe it well, it doesn't return a response that **contains** a string - it returns **a** string.

Again, follow the doc from python/node:
```
                If the value is successfully set, return OK.
                If value isn't set because of only_if_exists or only_if_does_not_exist conditions, return None.
                If return_old_value is set, return the old value as a string.